android/app/src/main/res/layout/feature_card.xml

```diff
@@ -5,15 +5,18 @@
     android:layout_width="match_parent"
     android:layout_height="wrap_content"
     android:layout_marginTop="14dp"
+    android:layout_marginLeft="16dp"
+    android:layout_marginRight="16dp"
+    android:background="@drawable/feature_card_border"
     android:orientation="vertical">
 
     <androidx.cardview.widget.CardView
         android:layout_width="match_parent"
         android:layout_height="wrap_content"
-        android:layout_marginLeft="16dp"
-        android:layout_marginRight="16dp"
+        android:layout_margin="@dimen/feature_card_border_width"
         android:layout_gravity="center_horizontal"
-        app:cardCornerRadius="5dp">
+        app:cardElevation="@dimen/feature_card_elevation"
```

Comment:
Nit: If `0` is the default (I think it was), I'd prefer to just not set this attribute.

Reply:
Default value is actually `2dp`.